common/lib/xmodule/xmodule/video_module/video_xfields.py

```diff
@@ -160,8 +160,8 @@ class VideoFields(object):
         default=False
     )
     edx_video_id = String(
-        help=_("If you were assigned a Video ID by edX for the video to play in this component, enter the ID here. In this case, do not enter values in the Default Video URL, the Video File URLs, and the YouTube ID fields. If you were not assigned an edX Video ID, enter values in those other fields and ignore this field."),
-        display_name=_("EdX Video ID"),
+        help=_("If you were assigned a Video ID by edX for the video to play in this component, enter the ID here. In this case, do not enter values in the Default Video URL, the Video File URLs, and the YouTube ID fields. If you were not assigned an Video ID, enter values in those other fields and ignore this field."),  # pylint: disable=line-too-long
```

Comment:
From @lamagnifica (Comment lost in last rebase)
@gwprice @BenjiLee @lou-wang how's this:
The single identifier for a video file that has multiple encodings hosted on multiple sites. To identify the locations of different versions of the video file, use either this field or the Default Video URL, Video File URLs, and YouTube ID fields.